rrrspec-client/lib/rrrspec/client/support.rb

```diff
@@ -40,7 +40,26 @@ def start_taskset(conf, rsync_name)
         return taskset
       end
 
-      def show_result(taskset, verbose=false)
+      def show_result(taskset, verbose=false, show_errors=false)
+        if show_errors
```

Comment:
I want to see this block https://github.com/sauliusgrigaitis/rrrspec/blob/show-errors/rrrspec-client/lib/rrrspec/client/support.rb#L63 at the very bottom, so I can quickly see is there any failing specs and how much specs failed. That's why I added code there.